dataprep/eda/correlation/compute.py

```diff
@@ -288,9 +279,27 @@ def pearson_nxn(data: da.Array) -> da.Array:
     """
     Pearson correlation calculation of a n x n correlation matrix for n columns
     """
-    cov = da.cov(data.T)
-    stderr = da.sqrt(da.diag(cov))
-    corrmat = cov / stderr[:, None] / stderr[None, :]
+    _, ncols = data.shape
```

Comment:
Can we handle the missing values before we hit the computation? It looks ugly if we mix the missing handle with computation.

Reply:
Of course, but it will pass two parameters: data(da.Array) and mask(da.Array). What's more, we also have to write for-loop to calculate the mask, which is very ugly. I admit that we need to optimize kendall_nxn and pearson_nxn function. However, I do not think it is ugly. I just reference this website: https://stackoverflow.com/questions/48455335/matlab-how-to-ignore-nan-values-in-the-corr-function to implement this function. If you have better solution, you could tell me as soon as possible.

Reply:
Ok!